app/components/steps/conclusion.js

```diff
@@ -0,0 +1,112 @@
+
+export default function Conclusion(props) {
+  const { className, discussionId, stepIntro, onDone = () => {}, ...otherProps } = props
+  const [data, setData] = useState({})
+  const classes = useStylesFromThemeFunction(props)
+
+  useEffect(() => {
+    window.socket.emit('get-conclusion', discussionId, data => {
+      if (data) {
+        const conclusion = data
+        setData(conclusion)
+      }
+    })
+  }, [])
+
+  const handleOnDone = ({ valid, value }) => {
+    window.socket.emit('upsert-jsform', discussionId, 'conclusion', { howDoYouFeel: value })
+
+    onDone({ valid })
+  }
+
+  return (
+    <div className={cx(classes.conclusion, className)} {...otherProps}>
+      <div className={cx(classes.mostAndLeastsWrapper)}>
+        <StepIntro {...stepIntro} />
+        <div>
+          <div className={cx(classes.subject)}>
+            <H>{data.point?.subject}</H>
+          </div>
+          <div className={cx(classes.description)}>
+            <text>{data.point?.description}</text>
+          </div>
+        </div>
+        <div className={cx(classes.dualPointList)}>
+          <ShowDualPointList leftPoints={data.mosts} leftHeader={"Why It's Most Important"} rightPoints={data.leasts} rightHeader={"Why It's Least Important"} vState={undefined} />
+        </div>
+      </div>
+      <div className={cx(classes.rankingResultsWrapper)}>
+        <div className={cx(classes.votingResultsText)}>
+          <text>Voting Results</text>
+        </div>
+        <div className={cx(classes.rankingResults)}>
+          <RankingResults
+            resultList={{
+              Most: data.mosts ? data.mosts.length : 0,
+              Neutral: 20,
+              Least: data.leasts ? data.leasts.length : 0,
+            }}
+          />
+        </div>
+      </div>
+      <div className={cx(classes.howDoYouFeelWrapper)}>
+        <HowDoYouFeel title={'How do you feel about the results?'} onDone={handleOnDone} />
+      </div>
+    </div>
+  )
+}
+
+const useStylesFromThemeFunction = createUseStyles(theme => ({
+  conclusion: {
+    background: theme.colorPrimary,
+    padding: '1rem',
+  },
+  subject: {
+    color: theme.colors.primaryButtonBlue,
+    fontSize: '3rem',
+    fontWeight: 700,
+    fontStyle: 'bold',
+    lineHeight: '2.9375rem',
+  },
+  description: {
+    color: theme.colors.title,
+    fontSize: '1.5rem',
+    fontWeight: 400,
+    lineHeight: '1.875rem',
+  },
+  dualPointList: {
+    paddingTop: '2rem',
+    paddingBottom: '10rem',
+  },
+  rankingResultsWrapper: {
+    backgroundColor: '#EBEBEBB2',
+    padding: '2rem',
+  },
+  rankingResults: {
+    backgroundColor: '#D9D9D999',
+    padding: '5rem',
+  },
+  votingResultsText: {
+    fontWeight: 300,
+    fontSize: '2.25rem',
+    lineHeight: '2.938rem',
+    padding: '2rem 0rem 3rem 2rem',
+    color: theme.colors.primaryButtonBlue,
+  },
+  howDoYouFeelWrapper: {
+    padding: '5rem 2rem 5rem 2rem',
+  },
+  mostAndLeastsWrapper: { padding: '0 4rem 0 4rem' },
```

Comment:
please eliminate the padding on mostAndLeastsWrapper so and let top most component handle the wrapping.